src/mc_tasks/lipm_stabilizer/StabilizerTask.cpp

```diff
@@ -443,8 +458,10 @@ void StabilizerTask::addToGUI(mc_rtc::gui::StateBuilder & gui)
                  NumberInput("Torso pitch [rad]", [this]() { return c_.torsoPitch; },
                              [this](double pitch) { c_.torsoPitch = pitch; }));
 
-  gui.addElement({"Tasks", name_, "Debug"}, Button("Disable stabilizer", [this]() { disable(); }),
-                 Button("Reconfigure", [this]() { reconfigure(); }), Button("Dump configuration", [this]() {
+  gui.addElement({"Tasks", name_, "Debug"}, ElementsStacking::Horizontal, Button("Disable", [this]() { disable(); }));
```

Comment:
`ElementsStacking::Horizontal` does not nothing if the elements are not added in the same call. Did you mean to add it with the other buttons?

Reply:
I initially had it stacked with the other buttons, but this wasn't very clear in the GUI, so I moved it by itself, and forgot to remove the stacking ;)